res/openvpn-gui-res-it.rc

```diff
@@ -260,19 +275,19 @@ END
 /* URL Profile Import Dialog */
 ID_DLG_URL_PROFILE_IMPORT DIALOG 6, 18, 249, 95
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER | DS_SETFOREGROUND
-CAPTION "Import Profile from Access Server"
+CAPTION "Importa profilo dal server di accesso"
```

Comment:
@bovirus This is marked as resolved in favour of retaining "Access Server" as per PR #486, but it seems you forgot to revert it. I can change this back during  merge if you are okay. Here and one instance below.

Reply:
OK but also here "Access Server" is the server type that provide the access and not a type of license for server.
OK I will change ita ss "Access ServeR" (also if I'm not agree).

Reply:
It is not a server type. It is literally a product name. That is like translating Microsoft Word and translating the "Word" part.

Reply:
Fixed.